content/messaging/agent-chat-api/index.mdx

```diff
@@ -187,7 +187,7 @@ File event informs about a file being uploaded.
 | `custom_id`                                           | optional  |                                                                   - |
 | `properties`                                          | optional  |                                                                   - |
 | `recipients`                                          | required  |                          Possible values: `all` (default), `agents` |
-| `system_message_type`                                 | required  |                                                                   - |
+| `url`                                                 | required  | Has to point to the LiveChat CDN.                                   |
```

Comment:
Does the url in the file event have to point to LC CDN?

Reply:
Yes. It's also recommended to use url returned by `upload_file`.